docs/index.md

```diff
@@ -87,6 +87,8 @@ Gravitino currently supports the following catalogs:
 * [**OceanBase catalog**](./jdbc-oceanbase-catalog.md)
 * [**StarRocks catalog**](./jdbc-starrocks-catalog.md)
 
+If you want to operate table and partition statistics, you can refer to the [document](./manage-statistics-in-gravitino.md).
```

Comment:
You should also add the link in this doc https://github.com/apache/gravitino/blob/891035d773a22ef7562cace75816c1a46494496b/docs/manage-relational-metadata-using-gravitino.md?plain=1#L1041

Reply:
OK, I added it.